packages/ui-components/src/components/toggle-button-group/toggle-button-group.tsx

```diff
@@ -0,0 +1,43 @@
+import { Component, h, Prop, Host, Event, EventEmitter } from '@stencil/core';
+import { IToggleButton } from '../toggle-button/toggle-button.types';
+import { IToggleButtonGroup, IToggleButtonGroupEvents } from './toggle-button-group.types';
+
+@Component({
+	tag: 'kv-toggle-button-group',
+	styleUrl: 'toggle-button-group.scss',
+	shadow: true
+})
+export class KvToggleButtonGroup implements IToggleButtonGroup, IToggleButtonGroupEvents {
+	/** @inheritdoc */
+	@Prop({ reflect: true }) buttons: IToggleButton[] = [];
+	/** @inheritdoc */
+	@Prop({ reflect: true }) selectedButtons?: Record<string, boolean> = {};
+	/** @inheritdoc */
+	@Prop({ reflect: true }) disabledButtons?: Record<string, boolean> = {};
```

Comment:
IMO, you should add a `withRadio` and `disabled` prop so we can easily add a radio button to all buttons/disable all buttons without the need to recreate an object. Furthermore, this prop should override the `disabledButtons` and `radioButtons` props values.

Reply:
Seems like sensible additions, I'll implement them :bulb: